src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -102,54 +104,48 @@ private boolean hasRemainingComments() {
       return lastCommentIndex < comments.size() - 1;
     }
 
+    private Comment getCurrentComment() {
+      return comments.get(lastCommentIndex);
+    }
+
+    private Comment getNextComment() {
+      return comments.get(lastCommentIndex + 1);
+    }
+
     /** Returns the ending line number of the current comment */
     private int getLastLineOfCurrentComment() {
-      return comments.get(lastCommentIndex).location.end.line + 1;
+      return getCurrentComment().location.end.line + 1;
     }
 
     /** Returns the starting line number of the next comment */
     private int getFirstLineOfNextComment() {
-      return comments.get(lastCommentIndex + 1).location.start.line + 1;
+      return getNextComment().location.start.line + 1;
     }
 
     /** Shifts a new comment into the group */
-    private void forceAddCommentToGroup() {
+    private void addNextCommentTo(List<Comment> commentGroup) {
```

Comment:
you only ever call this with `commentGroup == currentCommentGroup` - is that intentional? Would it be more readable to use the field then?